composer.json

```diff
@@ -19,14 +19,14 @@
         }
     ],
     "require":      {
-        "php": "^5.6 || ^7",
+        "php": "^7.1",
         "knplabs/knp-menu": "~2.3",
-        "symfony/framework-bundle": "~2.7|~3.0 | ^4.0"
+        "symfony/framework-bundle": "~2.7 | ~3.0 | ^4.0"
```

Comment:
Can you change this (and other relevant lines) to `^3.4 || ^4.0`?

Reply:
@garak done. Do you think I should also bump php version to 7.2 as 7.1 has ended its active support, or as it still receives security support is ok to leave at it is?
I'll vote for the first but I have really little experience in this kind of decisions when talking about OSS.

Reply:
I've also rebased on `master` in order to make all the tests pass

Reply:
@DonCallisto I see little advantage in excluding 7.1, we can think to raise requirements more in a next release